src/edfi-paging-test/edfi_paging_test/api/request_client.py

```diff
@@ -72,9 +73,30 @@ def _build_query_params_for_page(self, page_index, page_size: int) -> str:
         page_offset = (page_index - 1) * page_size
         return f"offset={page_offset}&limit={page_size}"
 
+    def _get_version(self) -> VersionInfo:
```

Comment:
I understand why you called it `_get-version`, but I wonder if there is another name, since retrieving a version number is not the actual goal here?